### DISCUSSION TITLE
NxNandManager_v5.0

### DISCUSSION THREAD

**Discussion by eliboa** [eliboa/NxNandManager#58] [General]
The following was changed since the last release:
- Added firmware detection support for 12.0.0, 12.0.1 & 12.0.2
- Added a feature to mount any FAT partition (USER, SYSTEM, etc.) as virtual disk. Follow steps to install Dokan's driver first.
![Gif](https://www.eliboa.com/NxNandManager_v5.0_mount.gif)
- Improved transfer speeds (especially for splitted NAND)
- Various bug fixes & improvements
   
Happy [May 17](https://may17.org/), celebrate the power of love

![GitHub Releases (by Release)](https://img.shields.io/github/downloads/eliboa/NxNandManager/v5.0/total)

<hr /><em>This discussion was created from the release <a href='https://github.com/eliboa/NxNandManager/releases/tag/v5.0'>NxNandManager_v5.0</a>.</em>

**Comment by mistervampi** [comment]
I just wanna thank you for this update and for your amazing work developing such amazing tool who for me has been a lifesaver for me and many consoles.

Also, the detail of May 17 is just 💓priceless!

**Comment by tenc3nt** [comment]
hi thanks for the release for v.5, i was load rawnand.bin with keyset, then i try to mount (USER, SYSTEM partitions), but it give error "cant assign a drive letter" am i do wrong step for install doakan driver? can u give any wiki/readme for mounting process on well procedure, thanks for ur hard work

**Comment by eliboa** [comment]
@tenc3nt It looks like the program can't found a valid mount point to mount virtual disk.
Next release will add the possibility for user to choose the mount point (i.e driver letter).

**Reply by tenc3nt** [reply]
yup, ur right, in my case i have lot of mount virtual disk (using raidrive/rclone), but not all of driver letter, cool thanks for respon, waiting for ur next release
